File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/points_goods_query_form.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PointsGoodsQueryForm:
    """积分商品查询请求"""
    page_num: int = None
    page_size: int = None
    sort_field: str = None
    sort_direction: str = None
    category: str = None
    min_points: int = None
    max_points: int = None
    exchangeable: bool = None
    size: int = None
    page: int = None

File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/PointsGoodsQueryForm.kt
package com.sdkwork.app

data class PointsGoodsQueryForm(
    val pageNum: Int? = null,
    val pageSize: Int? = null,
    val sortField: String? = null,
    val sortDirection: String? = null,
    val category: String? = null,
    val minPoints: Int? = null,
    val maxPoints: Int? = null,
    val exchangeable: Boolean? = null,
    val size: Int? = null,
    val page: Int? = null
)
